oneanddone/tests/functional/conftest.py

```diff
@@ -14,7 +16,18 @@ def capabilities(capabilities):
 
 @pytest.fixture
 def persona_test_user():
-    return requests.get('http://personatestuser.org/email').json()
+    max_retries = 5
+    attempt = 1
+    while attempt <= max_retries:
+        user_info = requests.get('http://personatestuser.org/email').json()
+        if user_info.get('email'):
```

Comment:
Do we only care about the 'email' key? I guess that will likely catch the majority of issues.